showcase/tests/integration/components/hds/accordion/index-test.gts

```diff
@@ -5,82 +5,99 @@
 
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'showcase/tests/helpers';
-import { click, render } from '@ember/test-helpers';
-import { hbs } from 'ember-cli-htmlbars';
+import { click, render, settled, find } from '@ember/test-helpers';
+import Accordion from '@hashicorp/design-system-components/components/hds/accordion/index';
+import Item from '@hashicorp/design-system-components/components/hds/accordion/item/index';
+import { tracked } from 'tracked-built-ins';
```

Comment:
ideally we would only use specific packages like TrackedArray, TrackedObject etc. from this package otherwise use `@glimmer/tracking`

as very soon we will migrate from `tracked-built-ins` to `@ember/reactivity/collections` https://rfcs.emberjs.com/id/1068-tracked-collections